diff --git a/node_modules/@smui/select/src/Option.svelte b/node_modules/@smui/select/src/Option.svelte
index 2e0638c..08255bf 100644
--- a/node_modules/@smui/select/src/Option.svelte
+++ b/node_modules/@smui/select/src/Option.svelte
@@ -55,7 +55,7 @@
   onDestroy(setSelectedText);
 
   function setSelectedText() {
-    if (selected && element) {
+    if (element && selected) {
       $selectedText = element.getPrimaryText();
     }
   }
